README.md

```diff
@@ -44,10 +48,6 @@ Retrieve a book from the API
 
 Entering `http://localhost:4741/books/1` into the chrome address bar.
 
-### Code Along: Browser Single Book
-
-Entering `http://localhost:4741/books/<integer>` into the chrome address bar.
-
```

Comment:
This is to demonstrate that we can make the request successfully from to the backend.